Dockerfile

```
@@ -3,10 +3,14 @@ FROM node:14.15.0-alpine
 WORKDIR /stremio
 
 ARG VERSION=master
+ENV FORCE_HTTPS=0
 
 RUN apk add --no-cache openssl-dev wget ffmpeg
 RUN wget https://dl.strem.io/four/${VERSION}/server.js
 RUN wget https://dl.strem.io/four/${VERSION}/stremio.asar
+ADD start.sh /stremio
+
+RUN ["chmod", "+x", "/stremio/start.sh"]
```

Rename this to:
```suggestion
RUN chmod +x start.sh
```
... for consistency with the other RUN statements. I don't think exec form is required for chmod to apply.